qemu/tests/cfg/numa_hmat.cfg

```diff
@@ -1,7 +1,7 @@
 - numa_hmat:
     type = numa_hmat
     only x86_64
```

Comment:
Thanks a lot for this update.
Enable HMAT on arm virt machine in the latest rebase.
Please help to add support for arm in cfg:
`only x86_64, aarch64`

And in with_node_initiator option, set to x86 only.

Reply:
@zhenyzha updated now, thanks!